docs/_template.html

```diff
@@ -33,8 +33,9 @@
       <div id="sidebar-logo" class="d-flex justify-content-center p-5">
       <a href="./index.html" ><img id="fsdocs-logo" src="{{root}}/{{fsdocs-logo-src}}" /></a>
 </div>
-      <div id="sidebar-generation" class="list-unstyled ps-0 text-white fs-6">
+      <div id=sidebar-content" class="list-unstyled ps-0 text-white fs-6">
```

Comment:
This was a tricky one, there is no opening `"` for the `id` attribute here.
This leads to the id not being set and introduces the menu colour change bug.

Reply:
that was fun, thanks!